lib/src/clojure_lsp/feature/code_actions.clj

```diff
@@ -155,6 +155,13 @@
              :command   "extract-function"
              :arguments [uri line character "new-function"]}})
 
+(defn ^:private extract-definition-action [uri line character]
+  {:title   "Extract definition"
```

Comment:
WDYT about `extract do def`? seems more clear to me what this code action will do

Reply:
Sounds good—that is much clearer.

Reply:
Done.